### DISCUSSION TITLE
About the report

### DISCUSSION THREAD

**Discussion by wanglu4042** [Xilinx/finn#851] [Q&A] [answered]
Hi, after I finished `model = model.transform(ZynqBuild(platform = test_pynq_board, period_ns = target_clk_ns))`
where can I check the total resources that in FPGA will be used?

![image](https://github.com/Xilinx/finn/assets/132076700/aedcf3fc-d7b9-48c0-83c4-30bc5e951904)

I did find many folders had a report, but I have no idea which would show the total used resources, like the LUT,BRAM......
Looking forward to your reply!
Thanks

**Comment by auphelia** [comment, marked as answer]
Hi @wanglu4042 , 

- When you use the builder the post synthesis resources will be saved in the output directory under the name `post_synth_resources.xml`.
- Additionally, your model contains the metadata property "vivado_synth_rpt" which contains the path to the report, you can access it with `model.get_metadata_prop("vivado_synth_rpt")`. 

From your overview you would need to get one level up and there is a file called synth_report.xml which gives you the post synthesis results. There is also a utilization report after placement under `impl_1/top_wrapper_utilization_placed.rpt` .

**Reply by kaycoemert** [reply]
How can I read the post_synth_resources.xml file? I have tried excel but it is hard to understand.
